golem_driving/envs/__init__.py

```diff
@@ -16,7 +16,7 @@ def add_env_args(parser: ArgumentParser) -> type(None):
     parser.add_argument('--domain-rand', action='store_true', help='enable domain randomization')
     parser.add_argument('--frame-skip', default=1, type=int, help='number of frames to skip')
     parser.add_argument('--frame-rate', default=None, help='number of frames per second')
-
+    parser.add_argument('--dir', default='./video', help='directory in which recordings are stored')
```

Comment:
I would prefer to pass it as config param, but to go on - you could change `default` to `Null` and later, if `dir is Null` do not wrap it in `Monitor`

Reply:
okay, I applied your suggestions.